cmd/livepeer/livepeer.go

```diff
@@ -148,8 +150,29 @@ func main() {
 		lpmon.Endpoint = *monhost
 	}
 	notifiee := bnet.NewBasicNotifiee(lpmon.Instance())
-	sourceMultiAddr, _ := ma.NewMultiaddr(fmt.Sprintf("/ip4/0.0.0.0/tcp/%d", *port))
-	node, err := bnet.NewNode([]ma.Multiaddr{sourceMultiAddr}, priv, pub, notifiee)
+	var maddrs []ma.Multiaddr
+	if *bindIPs != "" {
+		maddrs = make([]ma.Multiaddr, 0)
+		mas := strings.Split(*bindIPs, ",")
+		for _, m := range mas {
+			addr, err := ma.NewMultiaddr(m)
+			if err != nil {
+				glog.Errorf("Error creating bindIP %v to multiaddr: %v", m, err)
+				continue // nonfatal
+			}
+			maddrs = append(maddrs, addr)
+		}
+	} else if *transcoder {
```

Comment:
This bit is for after libp2p fixes their issue then transcoders can just have the correct IPs auto-selected.